From 014df16c3a367550ed7e7100d5331670489d0e9d Mon Sep 17 00:00:00 2001
Message-ID: <014df16c3a367550ed7e7100d5331670489d0e9d.1699403992.git.sharlatanus@gmail.com>
In-Reply-To: <0117df90b1ef43a67d4fa86e5b58e60f7c0a52e5.1699403992.git.sharlatanus@gmail.com>
References: <0117df90b1ef43a67d4fa86e5b58e60f7c0a52e5.1699403992.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 29 Oct 2023 15:20:32 +0000
Subject: [PATCH 41/67] gnu: go-github-com-goccy-go-json: Move to (gnu packages
 golang-web).

* gnu/packages/golang.scm (go-github-com-goccy-go-json): Move from
here...
* gnu/packages/golang-web.scm: ...to here.

Change-Id: I83fc749483f9f60a05c823e2f792efda0481b505
---
 gnu/packages/golang-web.scm | 22 ++++++++++++++++++++++
 gnu/packages/golang.scm     | 22 ----------------------
 2 files changed, 22 insertions(+), 22 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index 59312f791a..e92e04979e 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -8,6 +8,7 @@
 ;;; Copyright © 2020-2022 Efraim Flashner <efraim@flashner.co.il>
 ;;; Copyright © 2021 Raghav Gururajan <rg@raghavgururajan.name>
 ;;; Copyright © 2021 Sarah Morgensen <iskarian@mgsn.dev>
+;;; Copyright © 2022 Adam Kandur <kefironpremise@gmail.com>
 ;;; Copyright © 2022 Giacomo Leidi <goodoldpaul@autistici.org>
 ;;; Copyright © 2022 jgart via Guix-patches via <guix-patches@gnu.org>
 ;;; Copyright © 2022 muradm <mail@muradm.net>
@@ -358,6 +359,27 @@ (define-public go-github-com-go-chi-chi-v5
 decompose request handling into many smaller layers.")
     (license license:expat)))
 
+(define-public go-github-com-goccy-go-json
+  (package
+    (name "go-github-com-goccy-go-json")
+    (version "0.9.10")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/goccy/go-json")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "1bg8p4c6r8r0kixdxv2m8xmdsmb1zl5sd8czswpccldjk3c358wp"))))
+    (build-system go-build-system)
+    (arguments
+     '(#:import-path "github.com/goccy/go-json"))
+    (home-page "https://github.com/goccy/go-json")
+    (synopsis "JSON encoder/decoder in Go")
+    (description "Fast JSON encoder/decoder compatible with encoding/json for Go.")
+    (license license:expat)))
+
 (define-public go-github-com-google-safehtml
   (package
     (name "go-github-com-google-safehtml")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 4d504e30a2..5b37222c93 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -4002,28 +4002,6 @@ (define-public go-github-com-burntsushi-toml
 is similar to Go's standard library @code{json} and @code{xml} package.")
     (license license:expat)))
 
-(define-public go-github-com-goccy-go-json
-  (package
-    (name "go-github-com-goccy-go-json")
-    (version "0.9.10")
-    (source
-     (origin
-       (method git-fetch)
-       (uri (git-reference
-             (url "https://github.com/goccy/go-json")
-             (commit (string-append "v" version))))
-       (file-name (git-file-name name version))
-       (sha256
-        (base32 "1bg8p4c6r8r0kixdxv2m8xmdsmb1zl5sd8czswpccldjk3c358wp"))))
-    (build-system go-build-system)
-    (arguments
-     '(#:import-path "github.com/goccy/go-json"))
-    (home-page "https://github.com/goccy/go-json")
-    (synopsis "JSON encoder/decoder in Go")
-    (description
-     "Fast JSON encoder/decoder compatible with encoding/json for Go.")
-    (license license:expat)))
-
 (define-public go-github-com-getsentry-raven-go
   (let ((commit "5c24d5110e0e198d9ae16f1f3465366085001d92")
         (revision "0"))
-- 
2.41.0

